golem_sci/blockshelper.py

```diff
@@ -6,10 +6,11 @@ class BlocksHelper:
     def __init__(self, sci: SmartContractsInterface) -> None:
         self._sci = sci
 
-    def get_first_block_after(self, timestamp: int) -> Block:
+    def get_latest_existing_block_at(self, timestamp: int) -> Block:
         """
         Returns block with smallest number for which
-        `block.timestamp > timestamp` is satisfied.
+        `block.timestamp > timestamp` is satisfied or if
+        `block.timestamp < timestamp` then returns last block.
```

Comment:
It should rather read `or if such block doesn't exist returns latest block`

Reply:
Done